blueprints/addon/index.js

```diff
@@ -16,6 +16,10 @@ const stringifyAndNormalize = require('../../lib/utilities/stringify-and-normali
 module.exports = {
   description: 'The default blueprint for ember-cli addons.',
 
+  filesToRemove: [
```

Comment:
I'm not sure, but I think the `addon` blueprint only inherits the `files` folder, but nothing else.